demo/controllers/availableComponentsController.ts

```diff
@@ -0,0 +1,28 @@
+///
+/// Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+/// and other contributors as indicated by the @author tags.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
```

Comment:
No problem, I am not aware on the licensing model and we used it this way in hawkular, so I copy + pasted it to this repo as well.
It's true maintaining dates can be hard, so I'll remove these headers.